src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -240,7 +240,28 @@ internal static IntPtr NativeDllHandler(Assembly assembly, string libraryName)
         {
             s_nativeDllSubFolder ??= GetNativeDllSubFolderName(out s_nativeDllExtension);
             string folder = Path.GetDirectoryName(assembly.Location);
-            string fullName = Path.Combine(folder, s_nativeDllSubFolder, libraryName) + s_nativeDllExtension;
+
+            // Issue #20740 shows that some developers specify `.dll` in their DllImport attribute even though
```

Comment:
Please remove issue reference.
```suggestion
            // Some developers specify `.dll` in their DllImport attribute even though
```